xml/bura-start_stop_freezer_services.xml

```diff
@@ -47,17 +47,17 @@ ansible-playbook -i hosts/verb_hosts freezer-start.yml</screen>
    </step>
    <step>
     <para>
-     Run the following command to stop the freezer-agent:
+     Run the following command to stop the freezer agent:
     </para>
-<screen>sudo systemctl stop freezer-scheduler</screen>
+<screen>sudo systemctl stop openstack-freezer-scheduler.service</screen>
     <para>
      or run the following command to start the freezer-agent:
     </para>
-<screen>sudo systemctl start freezer-scheduler</screen>
+<screen>sudo systemctl start openstack-freezer-scheduler.service</screen>
     <para>
      or run the following command to restart the freezer-agent:
     </para>
-<screen>sudo systemctl restart freezer-scheduler</screen>
+<screen>sudo systemctl restart openstack-freezer-scheduler.service</screen>
```

Comment:
I think the service name is  openstack-freezer-scheduler.  sudo service openstack-freezer-scheduler works. sudo service openstack-freezer.scheduler.service could have problem. same comment applied to all above

Reply:
thanks! done